grails-app/views/robots.gsp

```diff
@@ -1,4 +1,4 @@
 <%@ page contentType="text/plain;charset=UTF-8" %>
 User-agent: *
 Allow: /
-Sitemap: https://portal.aodn.org.au/sitemap.xml
+Sitemap: ${grailsApplication.config.grails.serverURL.replaceFirst('/$',"")}/sitemap/sitemap.xml
```

Comment:
@pmbohm  We can easily tell the robot that it should not visit any pages on the site by using the following. I think this is a better place than chef. 

```
    <%@ page contentType="text/plain;charset=UTF-8" %>
    User-agent: *
<g:if env="production">
    Allow: /
    Sitemap: ${grailsApplication.config.grails.serverURL}sitemap/sitemap.xml
</g:if>
<g:else>
    Disallow: /
</g:else>
```